### DISCUSSION TITLE
Azure Active Directory is down

### DISCUSSION THREAD

**Discussion by bartosz-piotrowski-a** [AzureCommunityPL/group#248] [Pomoc]
Ktoś ma więcej informacji poza tym co jest w status page: 
https://status.azure.com/en-gb/status

Dzięki

**Comment by senionit** [comment]
Pytanie raczej powinniśmy inaczej zadać, czy ktoś z grupowiczów jest zaafektowany?

**Comment by maciejfrydrych** [comment]
Hmm z tego co mi wiadomo to każdy.
